cmd/mattermost/commands/command.go

```diff
@@ -67,3 +77,41 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func listCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+
+		return err
+	}
+	defer a.Shutdown()
+
+	var teams []*model.Team
+	if len(args) < 1 {
+		teamList, err := a.GetAllTeams()
+		teams = teamList
+		if err != nil {
+			return err
+		}
+	} else {
+		teams = getTeamsFromTeamArgs(a, args)
+	}
+
+	for i, team := range teams {
+		if team == nil {
+			CommandPrintErrorln("Unable to find team '" + args[i] + "'")
+			continue
+		}
+		result := <-a.Srv.Store.Command().GetByTeam(team.Id)
+		if result.Err != nil {
+			CommandPrintErrorln("Unable to list commands for '" + args[i] + "'")
+			continue
+		}
+		commands := result.Data.([]*model.Command)
+		for _, command := range commands {
+			commadList := fmt.Sprintf("%s: %s (team: %s)", command.Id, command.DisplayName, team.Name)
```

Comment:
Probably the name of the variable is confusing. maybe must be something like `commandListEntry` or `commandListItem`